src/constants/notification-content.js

```diff
@@ -89,25 +89,25 @@ export default function getNotificationContent({ type, payload }) {
 
     case "all_feedback_in": {
       fragments = [
-        'All volunteers of',
+        'All volunteers of ',
         {
           text: payload.project.name,
           url: '/projects/' + payload.project.id,
         },
-        'have submitted feedback',
+        ' have submitted feedback',
       ];
       break;
     }
 
     case "out_of_stock": {
       fragments = [
-        'The',
+        'The ',
         {
           text: (payload.product.type == null ?
             payload.product.name : `'${payload.product.name}' ${payload.product.type}`),
```

Comment:
This notification is when a specific variety goes out of stock. Maybe include it in the string?

Reply:
The crap thing is that there's no simple way to address a variety. Besides, I think it's best to make this notification product-level, not variety-level

Reply:
Why so? If size, put in between parentheses. If color, can also be put between parentheses, maybe with a bit of CSS coloring it.
Also, you mean when all varieties of the product go out of stock? So if there is one variety that almost no one wants and just a few of it left, but everything else is out of stock, the admin will still not get notified?